modules/eks/alb-controller-ingress-group/provider-kubernetes.tf

```diff
@@ -0,0 +1,132 @@
+##################
```

Comment:
since this file is a drop-in, make sure the same file exists / is updated under `mixins/` at the root of this repo. When we vendor this component downstream, the latest from `mixins/` will overwrite whatever is included here